Jenkinsfile

```diff
@@ -215,6 +215,70 @@ pipeline {
                 }
             }
         }
+        /************************************************************************
+         * STAGE
+         * -----
+         * Build Zowe SDK Bundle
+         *
+         * TIMEOUT
+         * -------
+         * 10 Minutes
+         *
+         * EXECUTION CONDITIONS
+         * --------------------
+         * - Always
+         *
+         * DECRIPTION
+         * ----------
+         * Gets the latest version of the Zowe SDK package from Zowe
+         * Artifactory. Creates an archive with 'fat' versions of the Plugins -
+         *  dependencies are bundled.
+         *
+         * OUTPUTS
+         * -------
+         * A Zowe CLI Plugins Archive containing Zowe CLI DB2 Plugin, Zowe CLI CICS Plugin,
+         * Zowe CLI z/OS FTP Plugin, Zowe CLI IMS Plugin, and Zowe CLI MQ Plugin.
+         ************************************************************************/
+        stage('Create Zowe CLI Plugins Bundle') {

```

Comment:
```
org.codehaus.groovy.control.MultipleCompilationErrorsException: startup failed:
WorkflowScript: 103: Duplicate stage name: "Create Zowe CLI Plugins Bundle" @ line 103, column 5.
       stages {
       ^
```

Reply:
big whoops!
Should be fixed now.
Also, added a TODO for the Python SDK